hepcrawl/extractors/jats.py

```diff
@@ -68,6 +68,16 @@ def _get_keywords(self, node):
                     free_keywords.append(keyword)
         return free_keywords, classification_numbers
 
+    def _get_public_notes(self, node):
+        """Return list of public notes from node."""
+        public_notes = []
+        for note in node.xpath('//notes[@notes-type]'):
+            for keyword in note.xpath('p/text()').extract():
+                public_notes.append({
+                    'value': keyword,
+                })
+        return public_notes
+
```

Comment:
Can you start working on that right away if you are blocked? it should not really affect the other prs (not much) but it will give us a lot of mind calm to be checking that throughout them.